fullscreen.bs

```diff
@@ -502,12 +520,19 @@ properties apply to this pseudo-element either.
 
 <h3 id=:fullscreen-pseudo-class><code>:fullscreen</code> pseudo-class</h3>
 
-<p>The <dfn id=css-pc-fullscreen><code>:fullscreen</code></dfn> pseudo-class must match any
-<a>element</a> that has its <a>fullscreen flag</a> set.
+<p>The <dfn id=css-pc-fullscreen><code>:fullscreen</code></dfn> pseudo-class must match an
+<a>element</a> <var>element</var> if any of the following conditions is true:
```

Comment:
"must match any element _element_ for which one of the following is true:" reads a little better I think.